Content.Client/Power/APC/UI/ApcMenu.xaml

```diff
@@ -0,0 +1,27 @@
+﻿<SS14Window xmlns="https://spacestation14.io"
+            Name="APCMenu"
+            Title="{Loc 'apc-menu-title'}"
+            Resizable="False">
+    <BoxContainer Orientation="Vertical"
+                  SeparationOverride="4">
+        <BoxContainer Orientation="Horizontal">
+            <Label Text="{Loc 'apc-menu-breaker-label'}"/><Label Text=" "/>
+            <Button Name="BreakerButton" Text="{Loc 'apc-menu-breaker-button'}"></Button>
+        </BoxContainer>
+        <BoxContainer Orientation="Horizontal">
+            <Label Text="{Loc 'apc-menu-external-label'}"/><Label Text=" "/>
+            <Label Name="ExternalPowerStateLabel" Text="{Loc 'apc-menu-power-state-good'}"/>
+        </BoxContainer>
+        <BoxContainer Orientation="Horizontal">
+            <Label Text="{Loc 'apc-menu-charge-label'}"/><Label Text=" "/>
```

Comment:
These `Label Text=" "/>`s should be removed and whitespace should be added to the end of the localization strings instead (using `{" "}`).